nautobot/docs/models/extras/relationship.md

```diff
@@ -97,7 +100,19 @@ From our many to many example above, we would use the following data to create t
 
 ### Configuring the Relationship between Objects
 
-Configuring the relationship is similarly easy. Send a request to `/extras/relationship-associations/` like the following:
+#### Via Object Endpoints
+
+_Added in version 1.4.0_ <!-- markdownlint-disable-line MD036 -->
```

Comment:
Nice. This is better than the disable/enable since it can be used inline.

Reply:
I think Bryan gets the credit on this one, but agreed it's nice :).